src/_scss/components/_pagination.scss

```diff
@@ -11,8 +11,8 @@
         @include flex-direction(row);
         float: right;
         margin: 0;
-        li {
-            button {
+        .pager__item {
```

Comment:
You need declare a block before declaring its member elements.

Reply:
`pager` is a block (buttons without the `# - ## of ## results` text)